components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -263,6 +263,13 @@ public enum ErrorMessages {
         ERROR_CODE_UNAUTHORIZED_FRAGMENT_APP_ACCESS("60066", "Fragment applications can be managed only " +
                 "from the organization that fragment application resides.", "Application %s can't be managed " +
                 "from organization %s."),
+        ERROR_CODE_INVALID_SHARE_APPLICATION_EMPTY_REQUEST_BODY("60067", "Invalid request.",
+                "At least one of the attributes from shareWithAllChildren and sharedOrganizations should be present."),
+        ERROR_CODE_INVALID_SHARE_APPLICATION_REQUEST_BODY("60068", "Invalid request.", "Cannot share " +
+                "the application with a set of child organizations when shareWithAllChildren is set."),
```

Comment:
Is this happens only when `shareWithAllChildren` is set to true. If so, let's mention it like `when shareWithAllChildren is set to true`.

Reply:
resolved with 2faf5c2e64b44d7a3e1145676bd4418471d360c1